android-sdk/templates/android-sdk-persistent-template.j2

```diff
@@ -132,7 +132,7 @@
                   }
                 ],
                 "terminationMessagePath": "/dev/termination-log",
-                "imagePullPolicy": "IfNotPresent",
+                "imagePullPolicy": "Always",
```

Comment:
This is useful for development, but is this OK in production environment ? Wdyt @laurafitzgerald @wei-lee ?